frontend/src/it/scala/bloop/CommunityBuild.scala

```diff
@@ -88,8 +88,8 @@ abstract class CommunityBuild(val buildpressHomeDir: AbsolutePath) {
 
   def loadStateForBuild(configDirectory: AbsolutePath, logger: Logger): State = {
     assert(configDirectory.exists, "Does not exist: " + configDirectory)
-    val loadedProjects = BuildLoader.loadSynchronously(configDirectory, logger)
-    val build = Build(configDirectory, loadedProjects)
+    val (loadedProjects, buildSettings) = BuildLoader.loadSynchronously(configDirectory, logger)
```

Comment:
Just nitpicking, but can we maintain here the order of "buildSettings, loadedProjects" ? It will then correspond to the `Build.apply` method